### ISSUE TITLE
fbjoin should check if you've already joined a given group chat

### ISSUE THREAD

**Issue by 00Davo** [jgeboski/bitlbee-facebook#33]
I hadn't used bitlbee's groupchat feature and wasn't sure how to get back into a groupchat after leaving it; just rerunning the `fbjoin` command *works*, but has the downside that it generates a fresh group channel in bitlbee every single time. The correct solution is to run `/join #channelname` to rejoin.

To avoid confusion of this sort, I suggest that on attempting to `fbjoin` a chat, bitlbee-facebook should first check whether that chat's already been joined, and if so, it should either

* implicitly `/join` the existing channel, as though the user had entered the correct command in the first place, or
* abort the `fbjoin` operation and produce an error message like "You're already in that groupchat - try /join #nameofchannel".

**Comment by ycny** [comment]
That's a good point. I also had to figure out how exactly fbjoin is working.

**Comment by maxux** [comment]
And, is there a way to auto-rejoins chatgroups after reconnection maybe ?

**Comment by jgeboski** [comment]
What am I missing?
```
<@jgeboski> fbjoin 1 testing
*chat is joined*
<@jgeboski> fbjoin 1 testing
<@root> facebook - Error: Failed to join chat: testing (640072176095558)
```

Also, you can just use `chan list`, and then `/join` the channel?

**Comment by ycny** [comment]
fbchats
    facebook - 1   my_name    user1, user2


    chan list
    0. &bitlbee
    1. #my_name

now, when you want to join channel my_name again:

    fbjoin facebook 1 my_name

and then:

    chan list
    0. &bitlbee
    1. #my_name
    2. #my_name_ <-- which is not what we could expect

I think the problem is that on page: https://wiki.bitlbee.org/HowtoFacebookMQTT there is a section: "Joining existing groupchats" which is misleading, because it can suggest joining already added groupchats in bitlbee, not existing one on real facebook server.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Also, you can just use chan list, and then /join the channel?

Yes. It is obvious now but as I said before, it was misleading to figure out how fbjoin is working so I'd suggest to change this tutorial on bitlbee wiki or add some annotation in bitlbee-facebook itself.

**Comment by dequis** [comment]
Well, I added a mention about this in the wiki.

>Note that fbjoin is only needed the first time you join a channel. After that, you can just do /join <channel>.
>
>You can configure these channels to be autojoined by doing channel <channel> set auto_join true

**Comment by jgeboski** [comment]
This has been resolved as of 073a946.
